test/builder-api/bin/cleanup-integration-tests.sh

```diff
@@ -77,7 +77,6 @@ do
   sql=$(cat <<EOF
 SET SEARCH_PATH TO shard_0;
 DELETE FROM busy_workers WHERE job_id IN (SELECT id FROM jobs WHERE project_name LIKE '$origin%');
-DELETE FROM graph_packages WHERE ident LIKE '$origin%';
```

Comment:
Why was this deleted?

Reply:
we deleted the table it was referencing https://github.com/habitat-sh/builder/pull/606/files#diff-5e7bf13e8c37dbad84f4d6524185aacbL332

Reply:
Ah, of course.  I missed that.